HISTORY.rst

```diff
@@ -2,9 +2,13 @@
 History
 =======
 
+1.0.0 (2023-05-18)
+------------------
+* First major release
```

Comment:
I suggest changing to "Start Beta version" or "Release Beta version" or something like this